dev-python/ciso8601/ciso8601-2.1.3.ebuild

```diff
@@ -0,0 +1,30 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+
+inherit distutils-r1
+
+DESCRIPTION="Fast ISO8601 date time parser for Python written in C"
+HOMEPAGE="https://github.com/closeio/ciso8601 https://pypi.org/project/ciso8601/"
+SRC_URI="mirror://pypi/${P:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~x86 ~amd64-linux ~x86-linux"
+IUSE="test"
+
+RDEPEND=""
+DEPEND="${REDEPEND}
+	dev-python/setuptools[${PYTHON_USEDEP}]
+	test? (
+		dev-python/nose[${PYTHON_USEDEP}]
+		dev-python/pytest[${PYTHON_USEDEP}]
+	)"
```

Comment:
If you use .zip files in SRC_URI you must add app-arch/unzip or it will fails to unzip (if someone does not have unzip). I suggest to use .tar.gz file version